src/otx/algo/modules/conv_module.py

```diff
@@ -359,3 +360,40 @@ class Conv3dModule(ConvModule):
     """A conv3d block that bundles conv/norm/activation layers."""
 
     _conv_nd = nn.Conv3d
+
+
+class Conv2dWithNormActivation(nn.Conv2d):
```

Comment:
You can use `Conv2dModule` instead by adding normalization and activation as arguments.

Reply:
I couldn't replace Conv2dWithNormActivation directly with Conv2dModule because it would break the weight loading mechanism. I implemented the custom weight loading to get around that in `MaskDINO._create_model.`